codingstandards/JS/README.md

```diff
@@ -0,0 +1,13 @@
+# GTMSportswear Javascript Coding Standards
+
+*GTMSportswear approach to Javascript*
+
+## Table of Contents
+
+1. [TypeScript Entry Points](https://github.com/GTMSportswear/docs/blob/master/codingstandards/JS/typescriptentrypoints.md)
```

Comment:
I would probably use a lowercase `js` for the folder name.

Reply:
@samiam236 I attempted to change the folder casing and it didn't register with git